source/docs/hardware/hardware-safety/io_safety.rst

```diff
@@ -0,0 +1,27 @@
+Input/Output Safety Mechanisms Built into the `FRC` Control System
+============================================================================
```

Comment:
Couple of issues here

- Underline is too long (legal in rST spec, but against our styleguide)
- You probably want \``FRC`` instead of FRC
- The title is too long, I think "Control System" and "FRC" is redundant